src/ddl/migrations/covid_hosp_facility_v0.3-v0.3.1.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO `covid_hosp_facility_key`
```

Comment:
This line is likely to fail, since when this file is run, the `covid_hosp_facility_key` table won't exist yet

Reply:
good catch, ill add the new table definition to this

Reply:
done